eng/testing/RunnerTemplate.sh

```diff
@@ -170,7 +170,12 @@ fi
 # ======================= BEGIN Core File Inspection =========================
 pushd $EXECUTION_DIR >/dev/null
 if [[ "$(uname -s)" == "Linux" && $test_exitcode -ne 0 ]]; then
-  echo Looking around for any Linux dump...
+  have_sleep=$(which sleep)
+  if [ -x "$have_sleep" ] ; then
```

Comment:
Should we do this only on helix machines? Maybe checking a helix env var?

Reply:
oh! How did I miss that.

Reply:
look correct now? I tested locally.